cg_hermes/config/balsamic.py

```diff
@@ -400,24 +400,24 @@
         "used_by": ["scout"],
     },
     frozenset({"vcf-filtered", "snv", "clinical-vcf-filtered"}): {
-        "tags": ["vcf-snv-clinical", "filtered"],
+        "tags": ["vcf", "vardict", "filtered"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["storage", "deliver"],
     },
     frozenset({"clinical-vcf-filtered-index", "vcf-filtered", "snv"}): {
-        "tags": ["vcf-snv-clinical-index", "filtered"],
+        "tags": ["vcf-index", "vardict", "filtered"],
```

Comment:
Same as previous comment. Can we remove `vardict` here? Right now `clinical-vcf-filtered` is VarDict, but it won't be in future.

Reply:
```suggestion
        "tags": ["vcf-index", "filtered", "vcf-snv-clinical"],
```

Reply:
Hehe I just added those based on @Mropat suggestion

Reply:
imho, we should avoid any specific names and words in the final delivery tags. Keeping them agnostic makes Hermes much more powerful.

e.g. Same tag can be used for Whole Genome and Whole Exome and Target-seq regardless of variant caller.

Reply:
We can not use the tag `vcf-snv-clinical` here as that tag indicates that the file should be uploaded to scout. There is only one file that can use that tag. Also this is a index file.